Wikimate.php

```diff
@@ -163,7 +163,7 @@ protected function token($type = self::TOKEN_DEFAULT)
 			return false;
 		}
 
-		$tokenResult = json_decode($response->body);
+		$tokenResult = json_decode($response->body, true);
```

Comment:
Strictly speaking, this change (and the similar one in login() ) isn't needed as the Result variables aren't used outside these methods. But mixing two different decoding styles in the class could be a bit confusing. So now they all use the associative array style.